src/tool_help.c

```diff
@@ -391,19 +392,19 @@ static const struct helptxt helptext[] = {
    "Specify request command to use"},
   {"    --request-target",
    "Specify the target for this request"},
-  {"    --resolve <host:port:address[,address]...>",
+  {"    --resolve <host:port:addr[,addr]...>",
    "Resolve the host+port to this address"},
   {"    --retry <num>",
    "Retry request if transient problems occur"},
+  {"    --retry-all-errors",
+   "Retry all errors (use with --retry)"},
   {"    --retry-connrefused",
    "Retry on connection refused (use with --retry)"},
   {"    --retry-delay <seconds>",
    "Wait time between retries"},
-  {"    --retry-all-errors",
-   "Retry all errors (use with --retry)"},
   {"    --retry-max-time <seconds>",
    "Retry only within this period"},
-  {"    --sasl-authzid <identity> ",
+  {"    --sasl-authzid",
```

Comment:
Ths option appears to accept an option, the manual page seems to be incomplete in this sense. Restore it and update the manual?
```c
      case '6': /* --sasl-authzid */
        GetStr(&config->sasl_authzid, nextarg);
        break;
```

Reply:
Same here